hazelcast-jet-core/src/main/java/com/hazelcast/jet/processor/Sources.java

```diff
@@ -44,6 +47,152 @@ private Sources() {
     }
 
     /**
+     * Convenience for {@link #streamMap(String, DistributedPredicate, DistributedFunction, boolean)}
+     * emitting output as a {@link EventJournalMapEvent}.
+     */
+    @Nonnull
+    public static ProcessorMetaSupplier streamMap(@Nonnull String mapName) {
+        return StreamEventJournalP.streamMap(mapName, null, null, false);
+    }
+
+    /**
+     * Returns a meta-supplier of processor that will stream the events
+     * of the Hazelcast {@code IMap} with the specified name.
+     * Given predicate and projection will be applied to the events on the source.
+     * <p>
+     * The processors will only access data local to the
```

Comment:
we have it for local streaming/reading from map/cache, if we are streaming/reading from a remote cluster it is irrelevant